### ISSUE TITLE
Text cut off at bottom when scrollable

### ISSUE THREAD

**Issue by levrik** [zeit/hyper#2468]
- [ ] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [x] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**: Windows 10 1709
- **Hyper.app version**: 2.1.1
- **Link of a [Gist](https://gist.github.com/) with the contents of your .hyper.js**: https://gist.github.com/levrik/ba854eeab882a9759f7f50b768ab4f6b
- **Relevant information from devtools** _(CMD+ALT+I on macOS, CTRL+SHIFT+I elsewhere)_: _N/A_
- **The issue is reproducible in vanilla Hyper.app**: yes

## Issue
![image](https://user-images.githubusercontent.com/9491603/32980014-971e3e6a-cc5f-11e7-8e85-81e7c02a4b36.png)

Does only happen when the scrollbar appears. I'm not able to scroll down further.
Makes it especially difficult with underscores in the last line since you cannot see them.

**Edit:** Starting from `canary.9` the whole last line is cut-off.

**Comment by Magneticmagnum** [comment]
Happens to me as well.  No plugins.
Sometimes, the entire bottom row doesn't scroll down, nor am I able to scroll further (Not able to see what I'm typing in the terminal)

**Comment by Nufflee** [comment]
Same here, when I run a command that fills up the whole screen, the text gets cut off - it doesn't autoscroll nor I can scroll myself. It's incredibly annoying.

**Comment by ineeve** [comment]
I can not replicate this issue. Can you give me any instructions on how to replicate it ?

**Comment by bennycooly** [comment]
I also have this issue. I'm on Windows running bash (WSL). Whenever the output of a command exceeds the bottom of the terminal, the text gets cut off and I can't even scroll or see the output. The only way to fix this is to issue a clear command.

**Comment by ineeve** [comment]
Can this be related to [https://github.com/xtermjs/xterm.js/issues/1136](https://github.com/xtermjs/xterm.js/issues/1136)?

**Comment by Ehekatl** [comment]
the fix for xtermjs is already merged https://github.com/xtermjs/xterm.js/pull/1157
any updates here?

**Comment by chabou** [comment]
the fix for xtermjs is included in our `2.0.0-canary.9`.

Maybe using xterm fit addon could help with this issue: https://github.com/zeit/hyper/pull/2594

**Comment by dougmaitelli** [comment]
Now sure if this is related to the same issue, but for me, I am never able to see the latest line on the console when I have scrollbar, the last line is always hid under the bottom / cut. Changing the padding did not help.

I have this problem since the older canary versions, and still have it on canary 9.

**Comment by levrik** [comment]
@dougmaitelli Having this issue now too. Before the last line was just a bit cut off, on the canary.9 version the whole line is cut off.

**Comment by Tyriar** [comment]
@chabou it's quite complex doing this yourself now due to the translation to and from "canvas pixels" which get weird with some zoom levels, you should adopt fit and report any issues you encounter.

**Comment by dougmaitelli** [comment]
This issue is no longer happening to me as this latest canary 10 release

**Comment by dliuzzi** [comment]
This is still occurring even on the latest canary.10 release (on Windows 10):

![hyper](https://user-images.githubusercontent.com/143884/35125326-8550df2e-fca9-11e7-8695-06056b682183.png)

When it happens I fix it by pressing F11 twice (going into fullscreen mode and out again.)

**Comment by phifa** [comment]
This bug seems still present. Anybody can confirm? Iam on 2.1.0 canary

**Comment by stu177** [comment]
Same, I still get this on Ubuntu 18.04 using Hyper 2.1.0-canary.2 (canary)

**Comment by jheidecker** [comment]
Same. 2.0.0 (stable)  Ubuntu 18.04

**Comment by AlexanderLJ** [comment]
Same here, 2.0.0 (Stable) Windows 10

**Comment by NerijusD** [comment]
Same here, 2.0.0 (Stable) Windows 10

**Comment by mikecousins** [comment]
Yup, this one is super annoying. Still exists on 2.1.0 on Windows 10.

**Comment by andymule** [comment]
Mine only does this when my buffer exceeds the vertical screen boundary after the first time I create a new tab (two visible). If i switch tabs or have more than two visible, the buffer is fixed. Having no visible tabs, my buffer seems to stop a little too high.

**Comment by Dragod** [comment]
this still happen to me as well,  win10
![image](https://user-images.githubusercontent.com/4368630/53429870-58b40e80-39e5-11e9-90e2-ce51fdcaca49.png)

**Comment by shiftgeist** [comment]
Happends to me as well (Hyper 2.1.2, Debian 9, Zsh)

**Comment by talentlessguy** [comment]
Windows: still happens (Windows 10, Debian WSL, Hyper 2.1.2)

**Comment by anthony-langford** [comment]
This is still happening on Windows 10 running Ubuntu WSL. A possible temporary fix is to change the zoom (Ctrl+- or Ctrl+=).

![hyper](https://user-images.githubusercontent.com/22820881/57545953-bfc93a00-7329-11e9-8e2f-60634cc1c55a.JPG)

**Comment by untrix** [comment]
Still happening with v 3.02 on macOS Mojave (on March 11, 2020).
![Screen Shot 2020-03-11 at 10 28 39 AM](https://user-images.githubusercontent.com/1545368/76445823-4522de00-6383-11ea-95bf-854c66f2be56.png)

![Screen Shot 2020-03-11 at 10 30 59 AM](https://user-images.githubusercontent.com/1545368/76445938-7c918a80-6383-11ea-9a31-cecee299a8a3.png)
